backend/src/config.ts

```diff
@@ -145,14 +158,23 @@ const defaults: IConfig = {
   },
   'SOCKS5PROXY': {
     'ENABLED': false,
+    'USE_ONION': true,
     'HOST': '127.0.0.1',
     'PORT': 9050,
     'USERNAME': '',
     'PASSWORD': ''
   },
   "PRICE_DATA_SERVER": {
     'TOR_URL': 'http://wizpriceje6q5tdrxkyiazsgu7irquiqjy2dptezqhrtu7l2qelqktid.onion/getAllMarketPrices',
-    'CLEARNET_URL': 'https://price.bisq.wiz.biz/getAllMarketPrices'
+    'CLEARNET_URL': 'https://price.bisq.wiz.biz/getAllMarketPrices',
+    'BISQ_URL': 'https://bisq.markets/api',
+    'BISQ_ONION': 'http://bisqmktse2cabavbr2xjq7xw3h6g5ottemo5rolfcwt6aly6tp5fdryd.onion/api'
+  },
+  "EXTERNAL_DATA_SERVER": {
+    'MEMPOOL_API': 'https://mempool.space/api/v1',
+    'MEMPOOL_ONION': 'http://mempoolhqx4isw62xs7abwphsq7ldayuidyx2v2oethdhhj6mlo2r6ad.onion/api/v1',
+    'LIQUID_API': 'https://liquid.network/api/v1',
+    'LIQUID_ONION': 'http://liquidmom47f6s3m53ebfxn47p76a6tlnxib3wp6deux7wuzotdr6cyd.onion/api/v1'
```

Comment:
@ayanamidev I think these should be called `MEMPOOL_CLEARNET`, `LIQUID_CLEARNET` and `BISQ_CLEARNET`, containing just the base URL, then `/api/v1` gets appended at the time you make the request.

Reply:
@knorrium In case that the configuration should be changed to connect localhost instance without the help of the nginx reverse proxy.

https://mempool.space/api/donations

Also, accessing /api/donations endpoint will result in `endpoint does not exist "/donations"` error.

https://mempool.space/api/v1/donations

While this one works well